docs/ruleset_functions/is_terminal_heat_source_none(terminal_b).md

```diff
@@ -0,0 +1,21 @@
+# is_terminal_heat_source_none   
+
+**Description:** Returns TRUE if the heat source associated with the terminal unit is None or Null. It returns FALSE if the terminal unit has a heat source other than None or Null.   
+
+**Inputs:**  
+- **B-RMR**: To evaluate if the heat source associated with the terminal unit is None or Null.   
+- **terminal_b.id**: The id of the terminal unit to evaluate.  
+
+**Returns:**  
+- **is_terminal_heat_source_none**: The function returns TRUE if the heat source associated with the terminal unit is None or Null. It returns FALSE if the terminal unit has a heat source other than None or Null.  
```

Comment:
I think this function should loop through all terminals served by a given terminal.served_by_heating_ventilation_air_conditioning_system and verify that heat source is none or null for all such terminals. That will also affect how this sub-function is called.

Reply:
i believe this is now looping through all terminal units attached to a baseline HVAC system.